extensions/default/src/Panels/Toolbox/ToolboxUI.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { PanelSection, ToolSettings, Tooltip } from '@ohif/ui';
+import classnames from 'classnames';
+
+function ToolboxUI(props) {
+  const { toolbarButtons, handleToolSelect, activeTool, toolOptions, numRows, servicesManager } =
+    props;
+  return (
+    <PanelSection title={'Segmentation Tools'}>
+      <div className="flex flex-col bg-black">
+        <div className="bg-primary-dark mt-0.5 flex flex-wrap py-2">
+          {toolbarButtons.map((toolDef, index) => {
+            if (!toolDef) {
+              return null;
+            }
+
+            const { id, Component, componentProps } = toolDef;
+            const isLastRow = Math.floor(index / 4) + 1 === numRows;
```

Comment:
Could you extract the 4 as a constant for now - at least that explains why the 4 is there, which I had to think about.